cores/esp32/main.cpp

```diff
@@ -3,6 +3,10 @@
 #include "esp_task_wdt.h"
 #include "Arduino.h"
 
+#ifndef LOOP_STACK_SIZE
```

Comment:
please rename to `CONFIG_ARDUINO_ LOOP_STACK_SIZE` so I can add this to the IDF Kconfig as well